### ISSUE TITLE
I reach 0.4056 in test set using fine tunning

### ISSUE THREAD

**Issue by vicdxxx** [joelthchao/tensorflow-finetune-flickr-style#11]
2018-07-02 08:16:08.890713 Iter 12160: Testing Accuracy = 0.4056

Why you are only  0.3250? So big  variance of training, what happened?

**Comment by joelthchao** [comment]
There was download issue couple months ago, therefore I switch to new one (should be bigger). The score I reported was using old dataset, hence the difference.
